### ISSUE TITLE
Qwen1.5-MoE

### ISSUE THREAD

**Issue by wuming123** [ollama/ollama#3410]
### What model would you like?

The code of Qwen1.5-MoE has been in the latest Hugging face transformers and we advise you to build from source with command , or you might encounter the following error:pip install git+https://github.com/huggingface/transformers

**Comment by easp** [comment]
Ollama doesn't use transformers. Support for Qwen MoE will have to come after llama.cpp supports it. There is a PR, but it looks like there is work to do before it can be merged. 
https://github.com/ggerganov/llama.cpp/pull/6074

**Comment by rick-github** [comment]
```
hf download --local-dir . Qwen/Qwen1.5-MoE-A2.7B-Chat
convert_hf_to_gguf.py . --outtype f16 --outfile Qwen1.5-MoE-A2.7B-Chat.gguf
echo FROM Qwen1.5-MoE-A2.7B-Chat.gguf > Modelfile
ollama create qwen1.5:7b-a2b-fp16
```
```console
$ ollama run qwen1.5:7b-a2b-fp16
>>> hello
Hello! How can I assist you today?

>>> why is the sky blue?
The sky appears blue to us because of a phenomenon called Rayleigh 
scattering. Sunlight, which consists of a spectrum of colors, enters Earth's 
atmosphere and encounters tiny molecules like nitrogen and oxygen atoms. As 
light travels through the atmosphere, these molecules scatter (or break up) 
the shorter wavelength colors (blue and violet) more than the longer 
wavelength colors (red, orange, yellow).

During sunset or sunrise, when the sun is closer to the horizon, the light 
has to travel through more of Earth's atmosphere to reach our eyes. This 
increased distance allows for more scattering, causing the blue and violet 
colors in sunlight to scatter in all directions. As a result, we see mostly 
red, orange, and yellow hues during these times because they are less 
scattered. The blue color is scattered in all directions but not as 
efficiently as the other colors, leading to the sky appearing bluer overall.

Moreover, at sunrise and sunset, the Earth's atmosphere becomes thinner, and 
light has to travel through more air molecules before reaching our eyes, 
which increases the amount of scattering. This increased scattering makes the 
blue color even more apparent in the sky during these times.

In summary, the sky appears blue because of Rayleigh scattering caused by the 
interaction of sunlight with Earth's atmosphere, where shorter wavelengths 
(blue and violet) are scattered more than longer wavelengths (red, orange, 
yellow).
```
